.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Is this needed? What does it do? I see our internal app codebase using single quotes but not have this prettier rule.

Reply:
Ah nvm, they pass it as an option in the binary instead: `/prettier --write --single-quote --no-semi 'src/**/*.js'`. I like putting it the config better 👍

Reply:
If we remove it `prettier` will use double quotes when formatting. It could also be in `.eslintrc` and `package.json` IIRC. I've put it in this file so editor plugins can use this setting.